packages/fxa-settings/src/components/PageTwoStepAuthentication/index.tsx

```diff
@@ -211,37 +240,55 @@ export const PageTwoStepAuthentication = (_: RouteComponentProps) => {
           <VerifiedSessionGuard onDismiss={goBack} onError={goBack} />
           {qrCodeUrl && showQrCode && (
             <>
-              <p className="my-4">
-                Scan this QR code using one of{' '}
-                <LinkExternal
-                  className="link-blue"
-                  href="https://support.mozilla.org/kb/secure-firefox-account-two-step-authentication"
-                >
-                  these authentication apps
-                </LinkExternal>
-                .
-              </p>
+              <Localized
+                id="tfa-scan-this-code"
+                elems={{
+                  linkExternal: (
+                    <LinkExternal
+                      className="link-blue"
+                      href="https://support.mozilla.org/kb/secure-firefox-account-two-step-authentication"
+                    >
+                      {' '}
```

Comment:
Just here to prevent a syntax error.  Actual content will be replaced.

Reply:
Heh this is exactly what is needed for https://github.com/mozilla/fxa/issues/7322